lib/oas_parser/definition.rb

```diff
@@ -49,9 +49,16 @@ def endpoints
     end
 
     def webhooks
-      return [] unless raw['webhooks']
-      raw['webhooks'].map do |name, definition|
-        OasParser::Webhook.new(self, name, definition)
+      if raw['webhooks']
```

Comment:
Can we change this to provide an early return of an empty array if neither `raw['webhooks']` or `raw['x-webhooks']` are present and also refactor the statement to separate contained `if` statements?

So something like:

`return [] unless raw['webhooks'] || raw['x-webhooks']`